### PR TITLE
new(cmd/driver): driver config namespace from env variable

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug
/kind cleanup

**Any specific area of the project related to this PR?**

/area cli

**What this PR does / why we need it**:

This PR enables the `namespace` parameter of `driver config` command to be passed as `FALCOCTL_DRIVER_CONFIG_NAMESPACE` env variable.

Also, it fixes k8s configMap commit code, switching from `Patch` to `Update` command.
Finally, logs are a little improved.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/driver/config/config.go [hunk 1]
```diff
@@ -108,12 +128,6 @@ func checkFalcoRunsWithDrivers(engineKind string) error {
 
 func (o *driverConfigOptions) replaceDriverTypeInFalcoConfig(driverType drivertype.DriverType) error {
 	falcoCfgFile := filepath.Clean(filepath.Join(string(os.PathSeparator), "etc", "falco", "falco.yaml"))
-	type engineCfg struct {
```
Comment (FedeDP) [inline cmd/driver/config/config.go:111]: Now used from both `replaceDriverTypeInFalcoConfig` and `replaceDriverTypeInK8SConfigMap`.

File: cmd/driver/config/config.go [hunk 2]
```diff
@@ -161,31 +175,76 @@ func (o *driverConfigOptions) replaceDriverTypeInK8SConfigMap(ctx context.Contex
...
-		Path:  "/data/" + configMapEngineKindKey,
-		Value: driverType.String(),
-	}}
-	plBytes, _ := json.Marshal(payload)
-
-	for i := 0; i < configMapList.Size(); i++ {
-		configMap := configMapList.Items[i]
-		currEngineKind := configMap.Data[configMapEngineKindKey]
-		if err = checkFalcoRunsWithDrivers(currEngineKind); err != nil {
+	updated := false
```
Comment (FedeDP) [inline cmd/driver/config/config.go:178]: Check that at least a configMap was updated, otherwise return error.

**Review by alacuku — APPROVED** [review]
That's nice!

/lgtm

**Pull request merged**